tests/ludwig/encoders/test_text_encoders.py

```diff
@@ -3,7 +3,61 @@
 
 from ludwig.encoders import text_encoders
 from tests.integration_tests.parameter_update_utils import check_module_parameters_updated
-from tests.integration_tests.utils import slow
+
+
+@pytest.mark.parametrize(
+    "encoder_cls",
+    [
+        text_encoders.ALBERTEncoder,
+        text_encoders.BERTEncoder,
+        text_encoders.XLMEncoder,
+        text_encoders.GPTEncoder,
+        text_encoders.RoBERTaEncoder,
+        text_encoders.GPT2Encoder,
+        text_encoders.DistilBERTEncoder,
+        text_encoders.TransformerXLEncoder,
+        text_encoders.CTRLEncoder,
+        text_encoders.CamemBERTEncoder,
+        text_encoders.MT5Encoder,
+        text_encoders.XLMRoBERTaEncoder,
+        text_encoders.LongformerEncoder,
+        text_encoders.ELECTRAEncoder,
+        text_encoders.FlauBERTEncoder,
+        text_encoders.T5Encoder,
+        text_encoders.XLNetEncoder,
+        text_encoders.DistilBERTEncoder,
+    ],
+)
+def test_hf_pretrained_default_exists(tmpdir, encoder_cls: text_encoders.HFTextEncoder):
+    """Test that the default pretrained model exists on the HuggingFace Hub.
+
+    This test merely checks that the default model name is valid. It does not check
+    the model end-to-end, as that would require downloading the model weights, which
+    can cause problems in the CI due to memory/runtime constraints.
+
+    TODO: add an end-to-end test for pretrained HF encoders.
+    """
+    from huggingface_hub import HfApi
+
+    hf_api = HfApi()
+    hf_api.model_info(encoder_cls.DEFAULT_MODEL_NAME)
+
+
+@pytest.mark.parametrize("pretrained_model_name_or_path", ["bert-base-uncased"])
+@pytest.mark.parametrize("reduce_output", [None, "sum", "cls_pooled"])
+@pytest.mark.parametrize("max_sequence_length", [20])
+def test_auto_transformer_encoder(
```

Comment:
Nice catch with the NoneType `AutoTransformerEncoder.DEFAULT_MODEL_NAME`. Could you add a short docstring explaining why this encoder has a separate test?

Reply:
done!

Reply:
Thanks!